### PR TITLE
Improve and cleanup core styles, improve seven skin

### PR DESCRIPTION
**Pull request by johnBartos**
- Fix issues with core styles
    - padding
    - colors
    - alignment
    - sizing
    - accessibility
- Delete redundant styling
- Improve seven skin

The core styles (i.e. the styling with no skin present) were ugly and had poor color contrast. This PR improves the core styling and mostly passes WCAG 2 & 3 accessibility standards. The seven skin was redesigned  n a monochromatic, slim, and accessible style.

JW7-2534

### CODE REVIEW

**Comment by robwalch** [comment]
Skip button looks too small like it lost all padding and rounded corners:
http://player-pr-test-jenkins.longtailvideo.com/builds/2220/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=skins_premium_skins

**Comment by dannyfinks** [comment]
dock button tooltips no longer have rounded corners

**Comment by robwalch** [comment]
@johnBartos Does this PR replace #1205? If so please close that one.

**Comment by robwalch** [comment]
@johnBartos please fix merge conflicts.

**Comment by dannyfinks** [comment]
Cuepoints markers in the time slider are not aligned properly - http://player-pr-test-jenkins.longtailvideo.com/builds/2220/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=cuepoints_thumbnails

**Comment by dannyfinks** [comment]
The control-bar in firefox does not stretch to the full region - any test page (Blocker)

**Comment by dannyfinks** [comment]
The volume scrubber in the Stormtrooper skin is not aligned properly

**Comment by dannyfinks** [comment]
Centered play button is too large on square shaped skins

**Comment by dannyfinks** [comment]
Volume Menu on Six Skin is broken

**Comment by dannyfinks** [comment]
Hover over styling of the centered play button in the Six/Glow skin are incorrectly applied.

**Comment by dannyfinks** [comment]
This may be slightly trivial, but the time-slider region in the Glow skin is difficult to distinguish

**Comment by dannyfinks** [comment]
Unable to see the background of the centered play button when no poster image is visible (black background)

File: src/css/imports/controlbar.less [hunk 1]
```diff
@@ -1,10 +1,8 @@
 @import "mixins";
+@import "vars";
 
 .jw-controlbar {
-    display: table;
     position: absolute;
-    right: 0;
-    left: 0;
```
Comment (robwalch) [inline src/css/imports/controlbar.less]: Removing `display: table;` could be why you are having trouble with the slider.

File: src/css/imports/controlbar.less [hunk 2]
```diff
@@ -1,10 +1,8 @@
 @import "mixins";
+@import "vars";
```
Comment (robwalch) [inline src/css/imports/controlbar.less]: Isn't vars imported in mixins?

File: src/css/imports/controlbar.less [hunk 3]
```diff
@@ -67,6 +68,7 @@
     line-height: @controlbar-height;
     vertical-align: middle;
     cursor: pointer;
+    padding: 0 @ui-padding;
```
Comment (robwalch) [inline src/css/imports/controlbar.less]: padding throwing things off?

File: src/css/skins/seven.less [hunk 4]
```diff
@@ -1,301 +1,67 @@
...
-    }
-
-    /* Position of remaining overlay containers */
-    .jw-menu,
-    .jw-volume-tip {
-        bottom: .24em;
-    }
-    
-    /* Skip icon and text styles */
-    .jw-skip {
```
Comment (robwalch) [inline src/css/skins/seven.less]: Looks like you removed the skip button customizations.

As a general rule. I expect that if you take something away from seven, you add it somewhere else, but I don't see that for skip.

**Comment by robwalch** [comment]
@dannyfinks how's this looking now?

File: src/css/imports/controlbar.less [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+//@import "vars";
 
 .jw-controlbar {
     display: table;
     position: absolute;
-    right: 0;
-    left: 0;
     bottom: 0;
     height: @controlbar-height;
+    width: 100%;
```
Comment (robwalch) [inline src/css/imports/controlbar.less]: Have you tested in ie9?

**Comment by robwalch** [comment]
I still like this "classic" responsive test page:

http://player-pr-test-jenkins.longtailvideo.com/builds/2298/archive/test/deprecated/options-stretching-responsive.html

File: src/css/skins/seven.less [hunk 6]
```diff
@@ -1,308 +1,85 @@
...
+    display: table;
+    border: 4px solid @accent-color;
+    font-size: 20px;
+  }
 
-        .jw-controlbar .jw-icon-next:before {
-            padding: 0 0.7em 0 0.125em;
-        }
+  .jw-display-icon-container > .jw-icon {
+    padding-left: 4px;
```
Comment (robwalch) [inline src/css/skins/seven.less]: This makes the replay and buffering icons look off-center. If it's intended to better align the play button only apply this to that icon or player state.

**Comment by robwalch** [comment]
The slider should reach the end when video is complete:

slider-not-at-end

**Comment by robwalch** [comment]
The skip icon in the off state should be the same color as the icons in the controlbar:

skip-icon-color-does-not-match-controlbar-icons-color-for-off-state

**Comment by dannyfinks** [comment]
Seems more difficult to seek now you need to be directly within the slider just for the cursor to change.

**Comment by dannyfinks** [comment]
There is some overlap with the overlay menus at the control-bar. @robwalch @esboyd any concern?

six_skin_overlap

**Comment by johnBartos** [comment]
screen shot 2016-06-30 at 4 25 15 pm

For my own posterity

**Comment by johnBartos** [comment]
screen shot 2016-07-05 at 10 57 34 am

tooltip is off center

**Comment by johnBartos** [comment]
The commit history is ghastly but rebasing is unusually difficult

**Comment by johnBartos** [comment]
test this please

**Comment by johnBartos** [comment]
test this please

**Pull request closed without merging**